flixel/group/FlxSpriteGroup.hx

```diff
@@ -280,6 +281,8 @@ class FlxTypedSpriteGroup<T:FlxSprite> extends FlxSprite
 		sprite.alpha *= alpha;
 		sprite.scrollFactor.copyFrom(scrollFactor);
 		sprite.cameras = _cameras; // _cameras instead of cameras because get_cameras() will not return null
+		
+		if (clipRect != null) clipRectTransform(sprite, clipRect);
```

Comment:
This null check now seems unnecessary, since `clipRectTransform()` can handle `null`?